content/automate/ManageIQ/System/Request.class/__methods__/dump_root.rb

```diff
@@ -0,0 +1,12 @@
+#
+# Description: Log all attributes stored in the $evm.root hash.
+#
+
+def dump_root
+  $evm.log("info", "Root:<$evm.root> Attributes - Begin")
+  $evm.root.attributes.sort.each { |k, v| $evm.log("info", "  Attribute - #{k}: #{v}") }
+  $evm.log("info", "Root:<$evm.root> Attributes - End")
+  $evm.log("info", "")
+end
+
+dump_root
```

Comment:
Since the goal is to use this as an embedded method we do not want the method to be called from here.  The automate method that embeds it would inject the call where needed.

Also need to consider moving it to another namespace in Automate where other common embedded methods would live.

cc @tinaafitz @mkanoor

Reply:
@billfitzgerald0120 Can we add a new class for these types of utilities in /System/CommonMethods/Utils?

Reply:
@billfitzgerald0120 The method should have the module information.
module ManageIQ
  module Automate
    module System
      module CommonMethods
        module Utils
          ........